index.html

```diff
@@ -34,5 +33,181 @@ <h1 class="header" >I’m Mariane<br>
+                    <p class="canopy">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                <div class="tags">
+                    <ul>
+                    <li class="tag1"><img src="image/Tag_html.svg" alt="HTML Tag"></li>
+                    <li class="tag2"><img src="image/Tag_css.svg" alt="HTML Tag"></li>
+                    <li class="tag1"><img src="image/Tag.svg" alt="HTML Tag"></li>
+                    </ul>
+                </div>
+            </article>
+                    <a class="nav-button" href=""><input type="button" class="mybutton" value="See Project"></a> 
+        </section>
+
+        <section class="border_section4"> 
+                <div class="image_section2">
+                    <img src="image/Snapshoot Portfolio.png" alt="Snapshoot Portfolio">
+                </div>
+                
+                <h2 class="tonic">Tonic</h2>
+                <div class="flex-container">
+                    <ul>
+                    <li class="blue">CANOPY</li>
+                    <li><img src="image/Counter.svg" alt="Counter"></li>
+                    <li class="blue">Back End Dev</li>
+                    <li><img src="image/Counter.svg" alt="Counter"></li>
+                    <li class="blue">2015</li>
+                    </ul>
+                </div>
+               
+                <article>
+                    <p class="canopy">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                <div class="tags">
+                    <ul>
+                    <li class="tag1"><img src="image/Tag_html.svg" alt="HTML Tag"></li>
+                    <li class="tag2"><img src="image/Tag_css.svg" alt="HTML Tag"></li>
+                    <li class="tag3"><img src="image/Tag.svg" alt="HTML Tag"></li>
+                    </ul>
+                </div>
+                </article>
+                    <a class="nav-button" href="#"><input type="button" class="mybutton" value="See Project"></a> 
+        </section>
+
+        <section class="border_section3">
+                <div class="image_section3">
+                    <img src="image/art_printing.svg" alt="Professional Art Printing">
+                </div>
+                <header class="border_heading">
+                    <h2 class="heading">Multi-Post<br>
+                        Stories</h2>
+                    <div class="flex-container">
+                        <ul>
+                        <li class="blue">CANOPY</li>
+                        <li><img src="image/Counter.svg" alt="Counter"></li>
+                        <li class="blue">Back End Dev</li>
+                        <li><img src="image/Counter.svg" alt="Counter"></li>
+                        <li class="blue">2015</li>
+                        </ul>
+                    </div>
+                </header>
```

Comment:
- [optional] kindly use Html tags in the appropriate places `header` is not part `section` 😍😍

Reply:
Thank you